Gemfile

```diff
@@ -1,11 +1,3 @@
 source 'https://rubygems.org'
 
-# For MiqPassword
-gem "manageiq-gems-pending", :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
-
-# For starting collector pods
-gem "kubeclient",  "~>4.0"
-# For reading from Authentications
-gem "pg",          "~>1.1"
-# For reading from Sources + Endpoints
-gem "rest-client", "~>2.0"
+gemspec
```

Comment:
Why make this a gem?

Reply:
It's much easier to get rspec and everything else set up.  We don't need to release it to rubygems.

Reply:
If it's an app it should not be a gem.  It's confusing.